.changeset/friendly-weeks-cry.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/create-app': patch
+---
+
+Rebind external route for catalog import plugin from `scaffolderPlugin.routes.root` to `catalogImportPlugin.routes.importPage`.
+
+To make this change to an existing app, make the following change to `packages/app/src/App.tsx`
+
+```diff
+const App = createApp({
+  ...
+  bindRoutes({ bind }) {
+    ...
+    bind(apiDocsPlugin.externalRoutes, {
+-     createComponent: scaffolderPlugin.routes.root,
++     registerComponent: catalogImportPlugin.routes.importPage,
```

Comment:
Just one more thought on this, that the API's have `kind: API` not `kind: Component` so maybe shall we call it `registerApi` instead?

Reply:
Yeah, makes sense. All changed now.